lib/node/runner/api_transaction.go

```diff
@@ -0,0 +1,91 @@
+package runner
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"boscoin.io/sebak/lib/block"
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/error"
+)
+
+const GetTransactionPattern string = "/transactions"
+
+func (nh NetworkHandlerNode) GetNodeTransactionsHandler(w http.ResponseWriter, r *http.Request) {
+	hashes := r.URL.Query()["hash"]
```

Comment:
As you know client API support url like this
`/transactions/{hash}`
At this point, you use a query instead. Is there any reason for that?
I know that it is not a big deal because the pattern does not needed to follow REST, just curious.

Reply:
@kfangw In proposed `Ballot` there can be up to `1000`(`MaxTransactionsInBallot`) transactions, so in theory the node should request up to `1000` transactions from proposer. To request huge list of `hash`, `/transactions/{hash}` is not acceptable, I think.

Reply:
Agreed.